### DISCUSSION TITLE
How to unsplit after vsplit or hsplit

### DISCUSSION THREAD

**Discussion by ecashin** [helix-editor/helix#4169] [General] [answered]
I've been looking for documentation on how to undo a vsplit, but I haven't found anything yet.  Is there a way (other than exiting)?

**Comment by kirawi** [comment, marked as answer]
`:quit`

**Reply by ecashin** [reply]
> (other than exiting)

**Reply by ecashin** [reply]
Turns out that `:quit` does not exit but does get rid of the current split pane.

Thanks, @kirawi .

**Comment by CptPotato** [comment]
There's also the window menu:

- <kbd>Space</kbd> <kbd>w</kbd> <kbd>q</kbd>: close the current window (not to be confused with `:wq`)
- <kbd>Space</kbd> <kbd>w</kbd> <kbd>o</kbd>: close all windows except the current one
